apps/calendar/test/unit/views/single_day_test.js

```diff
@@ -157,290 +163,302 @@ suite('Views.SingleDay', function() {
     });
 
-    assert.include(
-      daysHolder.innerHTML,
-      'is-partial is-partial-micro',
-      'smaller than 18min'
-    );
-
-    assert.include(
-      daysHolder.innerHTML,
-      'is-partial is-partial-tiny',
-      'between 18-30min'
-    );
-
-    assert.include(
-      daysHolder.innerHTML,
-      'is-partial is-partial-small',
-      'between 30-45min'
-    );
+    test('> should render', function() {
+      var data = {
+        amount: 3,
+        basic: [
+          makeRecord('Dolor Amet', '4:00', '6:00', 1),
+          makeRecord('Lorem Ipsum', '5:00', '6:00')
+        ],
+        allday: [
+          makeAlldayRecord('Curabitur')
+        ]
+      };
+      var makeFirstEventID = makeID.bind(this, subject, data.basic[0]);
+      var makeSecondEventID = makeID.bind(this, subject, data.basic[1]);
+      var makeAllDayEventID = makeID.bind(this, subject, data.allday[0]);
+
+      // notice that we are testing the method indirectly (the important thing
+      // to check is if the view updates when we receive new data)
+      subject._observer.write(data);
+
+      assert.lengthOf(
+        daysHolder.querySelectorAll('.md__event'), 2, '0: events'
+      );
+      assert.lengthOf(
+        alldaysHolder.querySelectorAll('.md__event'), 1, '0: all'
+      );
+
+      // testing the whole markup is enough to check if position and overlaps
+      // are handled properly and also makes sure all the data is passed to the
+      // templates, a drawback is that if we change the markup we need to update
+      // the tests (might catch differences that are not regressions)
+
+      var days = daysHolder.querySelectorAll('.md__day');
+      assert.lengthOf(days, 1, 'single day');
+      assert.equal(days[0].dataset.date, date.toString(), '[data-date]');
+
+      var busytimes = days[0].querySelectorAll('.md__event');
+      assert.lengthOf(busytimes, 2, 'busytimes');
+
+      // busytime #1
+      var busy = busytimes[0];
+
+      // position/size is enough to test overlaps
+      assert.equal(busy.style.height, '99.9px', '1: height');
+      assert.equal(busy.style.top, '200px', '1: top');
+      assert.equal(busy.style.width, '50%', '1: width');
+      assert.equal(busy.style.left, '50%', '1: left');
+      assert.include(
+        busy.href, '/event/show/' + data.basic[0].busytime._id, '1: href');
+
+      var labels = busy.getAttribute('aria-labelledby');
+      assert.include(labels, makeFirstEventID('title'), '1: label title');
+      assert.include(labels, makeFirstEventID('location'), '1: label location');
+      assert.include(labels, makeFirstEventID('icon'), '1: label icon');
+      assert.include(labels, makeFirstEventID('description'), '1: label desc');
+
+      assert.include(busy.className, 'has-alarms', '1: class');
+      assert.include(busy.className, 'has-overlaps', '1: class');
+
+      var remote = data.basic[0].event.remote;
+
+      var title = busy.querySelector('#' + makeFirstEventID('title'));
```

Comment:
`+ foo` uses less chars than `${foo}`.. I didn't change anything on these tests, just the indentation (since I grouped into a `suite`)